pkg/model/config/config_test.go

```diff
@@ -147,6 +147,39 @@ var _ = Describe("PluginConfig", func() {
 		Expect(config.DecodePluginConfig(key, &pluginConfig)).To(Succeed())
 		Expect(pluginConfig).To(Equal(expectedPluginConfig))
 	})
+
+	It("should Marshal and umarchal a plugin", func() {
+
+		By("Using config with extra fields as struct")
+		cfg := Config{
+			Version: Version3Alpha,
+			Plugins: PluginConfigs{
+				"plugin-x": map[string]interface{}{
+					"data-1": "plugin value 1",
+				},
+			},
+		}
+		b, err := cfg.Marshal()
+		Expect(err).To(BeNil())
```

Comment:
Usually `Expect(err).NotTo(HaveOccurred())` is used with `error` variables and `Expect(SomeFunc()).To(Succeed())` with direct function calls (they do exactly the same but they are easier to read like this). `Expect(err).To(BeNil())` would do also the same but it is less clear.
```suggestion
		Expect(err).NotTo(HaveOccurred())
```